src/pages/Trivia.js

```diff
@@ -1,11 +1,30 @@
 import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { TriviaHeader, Loading, Question } from '../components';
 
 class Trivia extends React.Component {
   render() {
+    const { questions, question: { qnNum } } = this.props;
+    const LENGTH_FIVE = 5;
+
+    if (questions.length !== LENGTH_FIVE) return <Loading />;
```

Comment:
Para as perguntas de verdadeiro e falso o tamanho do vetor é 5 também?
Acho que podemos testar e talvez alterar essa lógica para questions.lenght>=2.
Não tenho certeza se o risco existe ou se a solução apresentada vai resolver, mas acredito ser uma avaliação / teste necessária.

Reply:
Excelente consideração, mas acredito que o ideal é alterar para `questions.length === 0`. Pois necessitamos que para qualquer valor acima disso o componente `<Question />` seja renderizado. O que acha? :smiley: 

```suggestion
    if (questions.length === 0) return <Loading />;
```

Reply:
Concordo, a lógica fica até mais simples. Boa!!